content/outcomes/application-development/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Build Modern Applications
+weight: 1
+layout: single
+description: "Learn how we approach application development at VMware Tanzu Labs"
+team:
+  - VMware Tanzu Labs
+---
+ 
+## Why is this learning path important?
+ 
+Software engineering techniques have never evolved at such a quick pace as they are today. New frameworks, practices, and technologies keep emerging, while bringing new concepts and increased complexity of software. As a result, it feels hard to keep up with change and to make the right choices for your end-to-end modern application lifecycle.
+ 
+Beyond technology, this learning path also teaches the fundamentals of the 30+ year old [VMware Tanzu Labs](https://tanzu.vmware.com/labs) methodology on developing modern applications, including how to:
+ 
+- Build the right thing the first time.
+- Use Agile development to deliver the products your customers want.
+- Seed a new mindset. Quickly validate your concepts. Get them from idea to production in days, and not months.
+ 
+## Who is this Learning Path for?
+ 
+This learning path is primarily for software engineers and other technical development team members involved in the development of modern applications. 
+ 
+Managers, designers, and other non-technical stakeholders may also find this learning path useful for understanding new technical terminology and concepts. 
+ 
+## What you will learn
+ 
+You will learn everything needed to start as a newly onboarded team member. Each section in this learning path provides details on specific skills or technologies, and includes links to corresponding information. 
+ 
+The following topics are covered:
+ 
+* [Balanced Teams](/outcomes/application-development/balanced-teams/)
+* [Cloud Native Development](/outcomes/application-development/cloud-native-development/)
+* [Distributed, Remote, and Asynchronous Work](outcomes/application-development/remote/)
+* Pair Programming (coming soon)
+* Product Validation (coming soon)
+* Retrospectives (coming soon)
+* Test Driven Development (coming soon)
+* User Stories (coming soon)
+ 
+This learning path is also an adequate, on-the-clock preparation for getting the most out of an on-site or remote lab engagement.
+
+## Where can I share my feedback?
```

Comment:
Hmmm, if we want something like this it seems like we should solve it in a common place and not just on one of the outcome overview pages.

Reply:
Good point! Then we probably should remove this.

On the internal [cookbooks](https://appmod-cookbook.apps.pcfone.io/), we added an "Improve this page" link heading to every page.

Reply:
Done